### DISCUSSION TITLE
Unusable

### DISCUSSION THREAD

**Discussion by DancerEm** [community/community#139143] [Copilot Conversations]
### Select Topic Area

General

### Body

Hi, 
I subscribed to this and it’s not letting me have access as well as GitHub plus

**Comment by Akash1134** [comment]
Hi @, we're happy you're here! You are more likely to get a useful response if you are explicit about what your issue is, giving a few more details might help someone give you a nudge in the right direction. 

Are you getting message while accessing Copilot? Which IDE are you using?  

We recommend you follow the quickstart guide for GitHub Copilot while setting up GitHub Copilot on your machine. For more information, see "[Quickstart for GitHub Copilot](https://docs.github.com/en/copilot/quickstart)."

The GitHub Copilot extension is frequently updated to fix bugs and add new features. It's important to keep your extension up to date because older clients cannot communicate with the GitHub Copilot servers. Update your GitHub Copilot extension on all the machines you have it installed.

For more information about configuring GitHub Copilot in a supported IDE, see "[Configuring GitHub Copilot in your environment](https://docs.github.com/en/copilot/configuring-github-copilot/configuring-github-copilot-in-your-environment)."

**Comment by devKanix** [comment]
My github account was flaged how can i fix that?

**Reply by raushankumar1511** [reply]
you can use github discussion

**Reply by Akash1134** [reply]
The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by devKanix** [reply]
Okay thanks i'll try
